core/base/src/constants/chains.ts

```diff
@@ -49,6 +49,7 @@ const chainIdAndChainEntries = [
   [   57, "XRPLEVM"        ],
   [   58, "Plasma"         ],
   [   59, "CreditCoin"     ],
+  [ 20000, "HyperCore"     ],
```

Comment:
Should we move this to the bottom and leave a comment like "this is a placeholder, guardians are not actually connected to HyperCore"